### DISCUSSION TITLE
django.contrib.auth missing in INSTALLED_APPS?

### DISCUSSION THREAD

**Discussion by coding-beginner-1** [saleor/saleor-platform#239] [Q&A]
Hi friends,
I have installed the saleor-platform with docker on my windows computer. Everything works fine but I keep getting the same error when creating the superuser with:

```docker-compose run --rm api python3 manage.py createsuperuser```

Error:

```RuntimeError: Model class django.contrib.auth.models.Permission doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.```

Inserting `django.contrib.auth` into `INSTALLED_APPS` (saleor/saleor/settings.py) as proposed on Stackoverflow also doesn't work since there also is an app called `saleor.auth` and django doesn't seem to like 2 apps with the same name:

```
INSTALLED_APPS = [
    # External apps that need to go before django's
    "storages",
    # Django modules
    "django.contrib.contenttypes",
    "django.contrib.sites",
    "django.contrib.staticfiles",
    "django.contrib.postgres",
    "django_celery_beat",
    #"django.contrib.auth",
    # Local apps
    "saleor.permission",
    "saleor.auth",

Error: django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: auth
```

To solve this, I have tried to:

1. Removing saleor.auth from INSTALLED_APPS
2. Renaming saleor.auth to saleor.authentification and renaming the folder auth in the saleor directory to authentification

But nothing worked and I still can't create a superuser. I would appreciate any insight! Thank you! ;)

**Comment by fowczarek** [comment]
Some time ago we had a similar [issue #11753](https://github.com/saleor/saleor/issues/11753). 
Could you confirm which Saleor version you use?

**Reply by coding-beginner-1** [reply]
I used the saleor-platform git and downloaded it last week. I don't know which version that is.

However I found a workaround by not using the command from the documentation
`docker-compose run --rm api python3 manage.py createsuperuser`

but the command on the installation guide of saleor-platform
`docker-compose run --rm api python3 manage.py populatedb --createsuperuser`

I decided not to include Saleor into my project so I don't need a solution but it might be important for others.

**Reply by peelar** [reply]
Hi @johannesaschmid 👋 

Best of luck with your project 🤞. Would you mind sharing why you didn't go with Saleor? Can we do something better, or is it a matter of a lack of technical fit?

**Reply by coding-beginner-1** [reply]
Sure! I am new to programming so it would be really difficult for me to build a custom storefront for Saleor. I tried using the react-storefront but I found no solution to customize the design.
